bundles/org.openhab.binding.sonos/src/main/java/org/openhab/binding/sonos/internal/handler/ZonePlayerHandler.java

```diff
@@ -2028,15 +2028,15 @@ public void setRepeat(Command command) {
     }
 
     public void setSubwoofer(Command command) {
-        setEqualizerBooleanSetting(command, "SubEnabled");
+        setEqualizerBooleanSetting(command, "SubEnable");
     }
 
     public void setSubwooferGain(Command command) {
         setEqualizerNumericSetting(command, "SubGain", getSubwooferGain(), MIN_SUBWOOFER_GAIN, MAX_SUBWOOFER_GAIN);
     }
 
     public void setSurround(Command command) {
-        setEqualizerBooleanSetting(command, "SurroundEnabled");
+        setEqualizerBooleanSetting(command, "SurroundEnable");
```

Comment:
Shouldn't this be the same as what is expected in the SonosXmlParser?

Reply:
No and that is the reason of this quick fix.